assets/remote-template.html

```diff
@@ -551,15 +566,15 @@
                         <td style="font-size: 18px; font-family: Arial, Helvetica, sans-serif; color: #3f3f3f; text-align:left;
                           -ko-font-size: @[titleTextStyle.size]px; -ko-font-family: @titleTextStyle.face; -ko-color: @titleTextStyle.color">
                           <span style="color: #3f3f3f; -ko-color: @titleTextStyle.color" data-ko-editable="titleText">
-                          Title
+                          Cím
                           </span>
                         </td>
                       </tr>
                       <tr>
                         <td align="left" style="text-align: left; font-size: 13px; font-family: Arial, Helvetica, sans-serif; color: #3f3f3f;
                           -ko-font-size: @[longTextStyle.size]px; -ko-font-family: @longTextStyle.face; -ko-color: @longTextStyle.color"
                           data-ko-editable="longText" class="long-text links-color">
-                          <div>Far far away, behind the word mountains, far from the countries <a href="">Vokalia and Consonantia</a>, there live the blind texts. Separated they live in Bookmarksgrove right at the coast of the Semantics, a large language ocean. A small river named Duden flows by their place and supplies it with the necessary regelialia.</div>
+                            <div>Ide bármi leírható, ami érdekes lehet. Akár többször is. Viszont, ha felsorolást szeretnél használni, válaszd ki a blokkok közül a <strong>felsorolás</strong> blokkot.<br />Kellemes szerkesztést!</div>
```

Comment:
Are we sure about hungarian?

Reply:
Yes. It was the request.

Reply:
In that case OK.
It just feels strange...
Maybe later when we'll be experts at localization we can change this to english and add the hungarian translation ;)

Reply:
I think we can't use localization in the mosaico templates.